qemu/tests/timer_rtc_sync.py

```diff
@@ -94,6 +99,11 @@ def verify_timedrift(session, is_hardware=False):
 
     session = vm.wait_for_login(timeout=timeout)
 
+    error_context.context("Ensure host/guest is configured to maintain the RTC"
+                          " in universal time", test.log.info)
+    disable_local_rtc()
```

Comment:
Do we need to disable it before `sync host time with NTP server` ?

Reply:
I think so.

Reply:
`clock_sync_command` will stop the chronyd/ntp service, from https://access.redhat.com/solutions/3034801, you can see only if ntpd/chronyd is running, we need to do something, so if disable it before `sync host time with NTP server`, I am not sure it will be an issue.